src/Bridge/Symfony/Response/ErrorResponse.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Biig\Melodiia\Bridge\Symfony\Response;
+
+use Biig\Melodiia\Response\AbstractUserDataErrorResponse;
+use Biig\Melodiia\Response\Model\UserDataError;
+
+class ErrorResponse extends AbstractUserDataErrorResponse
+{
+    /** @var string[] */
+    private $errors;
+
+    /** @var string */
+    private $errorKey;
+
+    public function __construct(array $errors, string $errorKey = '')
+    {
+        $this->errors = $errors;
+        $this->errorKey = $errorKey;
+    }
+
+    /**
+     * @return UserDataError[]
+     */
+    public function getErrors(): array
+    {
+        $errors = new UserDataError($this->errorKey, []);
```

Comment:
Should be singular. Also I think it's better to create it in the constructor and store an array of `UserDataError`. (it allows us to add a method `addError()` in the future)

Also the attribute `errors` could be protected to allow extension.